src/cron/calculateTopReaders.sql

```diff
@@ -0,0 +1,75 @@
+WITH
+views AS MATERIALIZED (
```

Comment:
Do we need it materialized since its run only once?

Reply:
I think it improved the query time by a few seconds each run

Reply:
actually no, scratch that. I'd worsened it a bit. Will remove 🙈